jobs/JEVS_GLOBAL_ENS_WAVE_GRID2OBS_PLOTS

```diff
@@ -56,7 +56,7 @@ export VERIF_CASE=${VERIF_CASE:-grid2obs}
 # Define COMIN/COMOUT variables     
 ####################################
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
-export COMOUT=${COMOUT:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
+export COMOUT=${COMOUT:-$(compath.py -o $NET/${evs_ver})}/$STEP/$COMPONENT/$RUN.$VDATE
```

Comment:
@StevenSimon-NOAA , can you check this one? Why did `$STEP/$COMPONENT/$RUN.$VDATE` get added to the end of this definition of COMOUT? This could be a problem b/c in the driver upstream of this code, `dev/drivers/scripts/global_ens/plots/jevs_global_ens_wave_grid2obs_plots.sh` , the COMOUT definition ends at ${evs_ver}. See line 51:
```
export COMOUT=${OUTPUTROOT}/${NET}/${evs_ver}
```

Reply:
I added it because I thought it was consistent with the formatting of atmos plot j-jobs. I now see that this is not the case for the driver upstream.